### ISSUE TITLE
Repositioning the segment we left after the switch off 

### ISSUE THREAD

**Issue by marbeaupa** [MQZHot/ZSegmentedControl#1]
How it is possible to reposition the program on the left segment after the switch off of the program? I saw after the re-start of the program that the initial segment is always the same. Me too in one of my program I have this problem. I think we have to save in prefs the position and to write something in the Viewdidload.

**Comment by MQZHot** [comment]
Hi, Do you mean saving the current selected state? The next time you enter, the saved state is displayed ?

**Comment by marbeaupa** [comment]
Yes, in case you switch off the device during you are working, when you switch on again the device the segment must be the same you left before the switch off.

**Comment by MQZHot** [comment]
I think may be you need Userdefaults to control the on and off of the switch...or the condition how to determine of the switch
